tests/gnrc_sock_dns/README.md

```diff
@@ -0,0 +1,27 @@
+# Overview
+
+This folder contains a test application for RIOT's sock-based DNS client.
+
+# How to test with native
+
+Setup up a tap interface:
+
+    $ sudo ip tuntap add dev tap0 mode tap user $(id -u -n)
+    $ sudo ip a a 2001:db8::1/64 dev tap0
+    $ sudo ip link set up dev tap0
+
+Start dnsmasq (in another console):
+
+    $ sudo dnsmasq -d -2 -z -i tap0 -q \
+        --dhcp-range=::1,constructor:tap0,ra-only \
+        --listen-address 2001:db8::1 \
+        --host-record=example.org,2001:db8::1
```

Comment:
This is a pain to set-up on ubuntu (since the network manager already has its own `dnsmasq` instance running). I just killed the running `dnsmasq` instance, but I think that led to the problems pointed out below.